glib-macros/src/gerror_derive.rs

```diff
@@ -0,0 +1,68 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use proc_macro2::TokenStream;
+use proc_macro_error::abort_call_site;
+use quote::{quote, quote_spanned};
+use syn::{punctuated::Punctuated, spanned::Spanned, token::Comma, Data, Ident, Variant};
+
+use crate::utils::{crate_ident_new, parse_domain_name};
+
+// FIXME: merge with genum version
```

Comment:
Is that difficult? If not, maybe just do it here in a separate commit :)

Reply:
done, hopefully in a correct way